packages/zosfiles/__tests__/__system__/api/methods/rename/Rename.system.test.ts

```diff
@@ -80,19 +85,12 @@ describe("Rename", () => {
             });
         });
         describe("Failure Scenarios", () => {
-            afterEach(async () => {
-                try {
-                    await Delete.dataSet(REAL_SESSION, beforeDSName);
-                } catch (err) {
-                    Imperative.console.info(`Error: ${inspect(err)}`);
-                }
-            });
             it("Shouldn't be able to rename a data set that doesn't exist", async () => {
                 let error;
                 let response;
 
                 try {
-                    response = await Rename.dataSet(REAL_SESSION, "NON.EXISTING.SET", afterDSName);
+                    response = await Rename.dataSet(REAL_SESSION, "NON.EXISTING.SET", afterDataSetName);
```

Comment:
For future reference, try using a more unique dataset name (maybe "randomly" generated?)